core/app/models/spree/stock/estimator.rb

```diff
@@ -15,8 +20,14 @@ class OrderRequired < StandardError; end
       #   descending cost, with the least costly marked "selected"
       def shipping_rates(package, frontend_only = true)
         raise ShipmentRequired if package.shipment.nil?
-        raise OrderRequired if package.shipment.order.nil?
+        raise OrderRequired if package.shipment.order.nil? && @order.nil?
```

Comment:
Is there a reason for using `@order` instead of `order` here?

Reply:
Good spot, I think I used @order before adding the `attr_accessor` and didn't update it before pushing up, all fixed now,